### DISCUSSION TITLE
Path to tmp files

### DISCUSSION THREAD

**Discussion by trutimer** [adonisjs/core#1802] [Help] [answered]
Hello guys hope you good. I created a new project of api only and stored some pictures (locally) on tmp directory. I made several directories inside tmp directory and store some pictures there. I read that some people suggested I allow static middleware of which i did in kernel but still i cannot reach tmp subdirectories as it says Route not found.
Please help

**Comment by thetutlage** [comment, marked as answer]
Here's the answer https://github.com/adonisjs/core/discussions/1567#discussioncomment-63819. You need to create a custom route

**Reply by trutimer** [reply]
Thanks man, but How do I define "Application"?

**Reply by trutimer** [reply]
Oh! I got it, for anyone like me, I just changed the Appication.makePath() to Helpers.tmpPath() from the answer provided by @thetutlage since I used subdirectories in tmp. Thanks a lot man
